src/test/java/com/palantir/docker/proxy/DockerContainerInfoUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.proxy;
+
+import static org.junit.Assert.assertFalse;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.palantir.docker.compose.execution.DockerExecutable;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import org.junit.Test;
+
+public class DockerContainerInfoUtilsTest {
+    private static final String CONTAINER_ID = "container-id";
+    private static final String IP_FORMAT_STRING = "{{ range .NetworkSettings.Networks }}{{ .IPAddress }}{{ end }}";
```

Comment:
Can you move this to `DockerContainerInfoUtils` instead? Then use the same constant for both